src/main/resources/db/migration/common/V1_332__add_task_cleanup_current_event.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO darts.automated_task (aut_id,task_name,task_description,cron_expression,cron_editable, batch_size)
+VALUES (nextval('aut_seq'),'CleanupCurrentEvent','Cleans up the event','0 * 0/4 * * *',true,50);
```

Comment:
```suggestion
VALUES (nextval('aut_seq'),'CleanupEventCurrentFlag','Cleans up the event is_current flag','0 * 0/4 * * *',true,50);
```

Reply:
is this cron correct? for the `ApplyRetention` task which needs to run every hours we've used this expressions: ```
0 0 * * * *
``` 
which seems to be ok according to [here](https://stackoverflow.com/questions/70107416/running-cron-job-task-at-every-hour)

Reply:
ok i'll change wasnt sure about he correct cron expression to use